### DISCUSSION TITLE
Welcome to OpenCCU Discussions!

### DISCUSSION THREAD

**Discussion by jens-maus** [OpenCCU/OpenCCU#1007] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CryptoVaraan** [comment]
I am using windows 10 Home (x64), i wonder if i can run Raspberrymatic using Virtualbox and the supplied *.OVA file? Maybe this is not the place to ask but i am new to Raspberrymatic and don't know where i should ask (i bought the HmIP-RFUSB stick).

**Comment by jens-maus** [comment]
Please create a new topic in the Q&A section instead of adding your question here!
